README.md

```diff
@@ -6,25 +6,25 @@
 
 The Postgres operator manages PostgreSQL clusters on Kubernetes using the [operator pattern](https://coreos.com/blog/introducing-operators.html).
 During the initial run it registers the [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#customresourcedefinitions) for Postgres.
-The PostgreSQL CRD is essentially the schema that describes the contents of the manifests for deploying individual
-PostgreSQL clusters using StatefulSets and [Patroni](https://github.com/zalando/patroni).
+The `postgresql` CRD is essentially the schema that describes the contents of the manifests for deploying individual
+Postgres clusters using StatefulSets and [Patroni](https://github.com/zalando/patroni).
```

Comment:
Is this a general endorsement of "Postgres" over "PostgreSQL"?  I don't mind either way, but I'd love to have an explicit preference ;)

Reply:
I'd go Postgres everywhere (except when the actual CRD named `postgresql`, but we can also abbreviate to `pg`) just because it leaves less opportunities for typos.